CRM/Admin/Form/Job.php

```diff
@@ -131,6 +131,17 @@ public static function formRule($fields) {
       $errors['api_action'] = ts('Given API command is not defined.');
     }
 
+    $authorized = FALSE;
+    try {
+      $authorized = $apiKernel->runAuthorize($fields['api_entity'], $fields['api_action'], array('version' => 3) + $fields['parameters']);
+    }
+    catch (Exception $e) {
+      $authorized = FALSE;
+    }
+    if (!$authorized) {
+      $errors['api_entity'] = ts('You do not have permission to run this job please consult your Administrator');
+    }
```

Comment:
@monishdeb to my reading, the significance of the `catch (Exception $e)` is that *there was a programmatic or environmental error while checking authorization*. If it was just an expected with denied-access, then the `runAuthorize()` function should have returned FALSE.

Compare behaviors in the case where `runAuthorize()` executes normally and determines that authorization is *not allowed* (`$authorized == FALSE`):
 * In 10426, it would display the error. 
 * With your inlined sketch, it would not display the error.